src/main/java/practice/StreamPractice.java

```diff
@@ -14,7 +18,15 @@ public class StreamPractice {
      * "Can't get min value from list: < Here is our input 'numbers' >"
      */
     public int findMinEvenNumber(List<String> numbers) {
-        return 0;
+        return numbers
+                .stream()
```

Comment:
It is better to leave the stream() at the same level as the variable for which we are calling
```suggestion
        return numbers.stream()
```